llarp/handlers/tun.cpp

```diff
@@ -168,86 +164,12 @@ namespace llarp
       }
        */
 
-#ifdef LOKINET_EXITS
-
-      if (not conf.m_exitNode.empty())
-      {
-        IPRange exitRange;
-        llarp::RouterID exitRouter;
-        std::string routerStr;
-        const auto pos = conf.m_exitNode.find(",");
```

Comment:
I have a `split()` function in `util/str.hpp` which may not be quite as efficient but cuts down on a lot of verbose string parsing code (it uses string_view, but does allocate for `std::vector`)